draft-polli-resource-digests-http.md

```diff
@@ -600,16 +638,45 @@ Response:
 
 # Security Considerations
 
+## Broken cryptographic algorithms
+
+Cryptographic algorithms are supposed to provide a proof of integrity
+usable eg. in signatures.
+
+To discourage mistakes, broken-cryptographic algorithms
+like `MD5` and `SHA-1` are now NOT RECOMMENDED.
```

Comment:
I would mofidy these two paragraphs a little and say
"Cryptogrphic alorithms are intended to provide a proof of integrity suited towards cryptographic constructions such as signatures. However, these rely on collision-resistance for their security proofs [CMU-836068]. The `MD5` and `SHA-1` algorithms are vulnerable to collisions attacks and they are NOT RECOMMENDED.